src/commands/misc.ts

```diff
@@ -38,36 +49,24 @@ export class MiscCommand {
                 { name: "Misc", value: '', inline: true }
             ])
         message.channel.send(embed);
-    }
-
-    static sodbotReplies = [
```

Comment:
FYI I fixed this by removing the global scope you assigned here (and putting it back in static). The problem with static stuff is it needs to be accesssed with the class name rather then this. (which was my mistake down at line 63.) Fixed that by changing `this` to `MiscCommand`